src/software/test_util/path_planning_test_util.h

```diff
@@ -41,4 +41,13 @@ namespace TestUtil
      */
     void checkPathDoesNotIntersectObstacle(std::vector<Point> path_points,
                                            std::vector<Polygon> obstacles);
+
+    /**
+     * Given a list of points, print it out to the console. ONLY mean to be used as a
+     * debugging tool.
```

Comment:
nit: you can remove the `ONLY mean to be used as a debugging tool`. Its in test_util so its implied

Reply:
fixed